app/controllers/miq_task_controller.rb

```diff
@@ -394,32 +385,40 @@ def build_query_for_warn
   end
 
   def build_query_for_status_completed(status)
-    return ["(state=? AND status=?)", ["finished", status]] if vm_analysis_task?
-    ["(state=? AND status=?)", ["Finished", status.capitalize]]
+    return ["(#{db_table}state=? AND #{db_table}status=?)", ["finished", status]] if vm_analysis_task?
```

Comment:
I like the way you formatted this in `build_query_for_running`
would it make sense to do that change elsewhere?

(Or would it make sense to change that other one back - to introduce less change?)